vision/dcgan_mnist/dcgan_mnist.jl

```diff
@@ -30,12 +30,16 @@ function create_output_image(gen, fixed_noise, hparams)
     return image_array
 end
 
+  
+# weight initialization as given in the "4.DETAILS OF ADVERSARIAL TRAINING" section of the paper
```

Comment:
Can you include a link to the paper (maybe on arXiv?)

Reply:
Yes will do